dandi/metadata.py

```diff
@@ -150,7 +153,7 @@ def get_metadata(
                 __import__(import_mod)
 
         meta["nd_types"] = get_neurodata_types(path)
-    else:
+    if not meta:
         raise RuntimeError("Unable to get metadata from non-BIDS, non-NWB asset.")
```

Comment:
might be useful to include path to the asset in the message to not make user guess

Reply:
including path in output here [7447000](https://github.com/dandi/dandi-cli/pull/1183/commits/74470007fe92630a381c4fcac7b073149336210c).